### ISSUE TITLE
podman container status switches between `initialized` and `Exited (17) 2 seconds ago`

### ISSUE THREAD

**Issue by DaRandomCube** [winapps-org/winapps#322]
since stock windows 10 was hogging my space, i decided to redo everything using tiny10 image
it all fun and game, unless i try to start the container after install, it doesn't start
here is the output of ` podman-compose --file ~/.config/winapps/compose.yaml ps`, sometimes shows this
```
CONTAINER ID  IMAGE                             COMMAND     CREATED         STATUS       PORTS                                                                                       NAMES
8e1ccda11d48  docker.io/dockurr/windows:latest              11 minutes ago  Initialized  0.0.0.0:3389->3389/udp, 0.0.0.0:3389->3389/tcp, 0.0.0.0:8006->8006/tcp, 3389/tcp, 8006/tcp  WinApps

```

and sometimes this

```
CONTAINER ID  IMAGE                             COMMAND     CREATED         STATUS                     PORTS                                                                                       NAMES
8e1ccda11d48  docker.io/dockurr/windows:latest              15 minutes ago  Exited (17) 2 seconds ago  0.0.0.0:3389->3389/udp, 0.0.0.0:3389->3389/tcp, 0.0.0.0:8006->8006/tcp, 3389/tcp, 8006/tcp  WinApps

```
made sure that the ports aren't blocked by a firewall, nothing helped

**Comment by DaRandomCube** [comment]
Any updates?

**Comment by oskardotglobal** [comment]
This is a duplicate of #251

**Comment by DaRandomCube** [comment]
> This is a duplicate of #251

unrelated
only happens when there is no enough memory, after closing some apps in the background, it will work fine
